test/integration/apiserver/cel/validatingadmissionpolicy_test.go

```diff
@@ -502,26 +522,53 @@ func Test_ValidateAnnotationsAndWarnings(t *testing.T) {
 // Test_ValidateNamespace_WithConfigMapParams tests a ValidatingAdmissionPolicy that validates creation of a Namespace,
 // using ConfigMap as a param reference.
 func Test_ValidateNamespace_WithConfigMapParams(t *testing.T) {
+	resetPolicyRefreshInterval := generic.SetPolicyRefreshIntervalForTests(policyRefreshInterval)
+	defer resetPolicyRefreshInterval()
+	featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, genericfeatures.ValidatingAdmissionPolicy, true)
```

Comment:
Note: We can drop these featuregate checks any time, now that the feature is locked to enabled for GA. Separate PR is probably better.

Reply:
remove featuregate ValidatingAdmissionPolicy PR: https://github.com/kubernetes/kubernetes/pull/126645